contracts/apps/disputable/IDisputable.sol

```diff
@@ -36,9 +37,7 @@ contract IDisputable is ERC165 {
     * @param _interfaceId The interface identifier being queried, as specified in ERC-165
     * @return True if the contract implements the requested interface and if its not 0xffffffff, false otherwise
     */
-    function supportsInterface(bytes4 _interfaceId) external pure returns (bool) {
-        return _interfaceId == DISPUTABLE_INTERFACE_ID || _interfaceId == ERC165_INTERFACE_ID;
+    function supportsInterface(bytes4 _interfaceId) public pure returns (bool) {
```

Comment:
Also wondering if this should be in the `Disputable` implementation

Reply:
But, what’s the reason? If you declare a contract to be `IDisputable`, then that contract will implement Disputable Interface, by definition (and ERC165 too). So the behavior of this function will be correct. If you are able to trick the compiler in a way that I can’t imagine, or for some reason that I can’t imagine either you want to hide that your contract implements those interfaces, you can always override the one here. But for regular cases you already have this “bureaucratic” function implemented in the base, so you don’t have to bother with it.
What’s the advantage of removing it from here?

Reply:
My main rationale was to align it with `IAragonApp`, which leaves its ERC-165 `supportsInterface()` out of the interface contract. Rather than include it implicitly, I'd prefer to leave such things to be done in the same, and obvious, manner at the implementation site, even if it's not DRY.

Also note that interface files are usually meant to be exported and copied around, and usually these do not include methods from ERC-165, as this is a separate interface.

However, I'm seeing that `IArbitrable` is also defining it inside of the interface, but `IArbitrator` does not even declare an ERC-165 interface at all.